### DISCUSSION TITLE
Laravel 11 | Table prefix, subdomain

### DISCUSSION THREAD

**Discussion by levaweb** [austintoddj/canvas#1429] [General]
Option 1: My laravel 11  entry-point is in /public/index.php, can I install Canvas in a subfolder of /public sharing the _same database_ but adding a _prefix_ to Canvas tables? (eg can_)

/public_html/laravel
       /app
      /public/blog
      ... 

Option 2: if #1 is not feasible, I can install Canvas in a _subdomain_ under a _new laravel_ instance _and_ new database.
/public_html/blog

Please advise.
Many thanks

https://github.com/austintoddj/canvas/discussions/949

**Comment by austintoddj** [comment]
Canvas still expects to live in your Laravel app (same DB). Tables are `canvas_*` by design — there’s no first-class table-prefix config.

Subdomain for the **admin SPA** is supported via `CANVAS_DOMAIN` / `config('canvas.domain')`. A fully separate Laravel app + DB for the blog remains a host architecture choice, not something Canvas automates.
